modules/psap-special-resource-operator-using.adoc

```diff
@@ -0,0 +1,378 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="using-the-special-resource-operator"]
+= Using the Special Resource Operator
+
+As an example, the Special Resource Operator can be used to manage the build and deployment of a driver container for a minimal kernel module called simple-kmod. 
+
+The objects required to build and deploy the kernel module can be defined using a Helm chart. The SRO image contains a local repository of Helm charts including the templates for deploying the simple-kmod. SRO can also read the templates from a ConfigMap. Both methods are demonstrated in the following examples.
+
+[id="deploy-simple-kmod-using-local-chart"]
+== Build and run the simple-kmod SpecialResource using the SRO image's local manifests.
+
+.Prerequisites
+
+* You have a running {product-title} cluster
+* You have installed the OpenShift CLI (`oc`).
+* You are logged into the OpenShift CLI as a user with `cluster-admin` privileges.
+* You have installed the xref:../psap-node-feature-discovery#installing-the-node-feature-discovery-operator_node-feature-discovery-operator[Node Feature Discovery (NFD) Operator]
+* You have installed the Special Resource Operator.
+
+.Procedure
+. To deploy the simple-kmod using the SRO image's local Helm repository, the following SpecialResource manifest can be used. Save this YAML as `simple-kmod-local.yaml`
```

Comment:
1)
s/the following SpecialResource manifest can be used./use the following SpecialResource manifest./

If this is more of a suggestion than a directive, maybe change it to:   you can use the following SpecialResource manifest.

2)
Also, please add a period to the end of the second sentence.